src/index.js

```diff
@@ -1,12 +1,51 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
-
-ReactDOM.render(<App />, document.getElementById('root'));
-
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+import React, { useState, useRef, useEffect } from "react";
+import ReactDOM from "react-dom";
+import { PlayCircleTwoTone } from '@ant-design/icons';
+import "./styles.css";
+
+function App() {
+  const audioPlayer = useRef(null);
+  const [progress, setProgress] = useState(80);
+
+  useEffect(() => {
+    console.log(progress, audioPlayer.current.duration);
+  }, [progress]);
+  const [isPlay, setPlay] = useState(true);
```

Comment:
Recomendaría que todas las declaraciones de estado vayan al principio, esto permite ver más fácilmente cómo está hecho el componente.
También, es una pavada pero, se entendería más si pones algo como:
```suggestion
  const [isPlaying, setPlaying] = useState(true);
```
Ya que la traducción literal de `isPlaying` sería: "Se está reproduciendo"